docs/programs/anchor/client-typescript.md

```diff
@@ -0,0 +1,356 @@
+---
+title: JS/TS Client
```

Comment:
These titles and descriptions should be more action oriented. Something like "Working with programs with Typescript". Same with IDLs and structure